lib/github-pages-health-check/resolver.rb

```diff
@@ -6,7 +6,8 @@ class Resolver
       DEFAULT_RESOLVER_OPTIONS = {
         :retry_times   => 2,
         :query_timeout => 5,
-        :dnssec        => false
+        :dnssec        => false,
+        :do_caching    => false
```

Comment:
Obviously nothing we can do about it, but just wanted to note that `do_caching` is a silly flag name. :smile: